imagersite/imager_images/models.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import unicode_literals
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext as _
+import uuid
+
+# Create your models here.
+
+@python_2_unicode_compatible
+class Photo(models.Model):
+
+    PRIVATE, SHARED, PUBLIC = 'Pri', 'Shr', 'Pub'
+
+    PUBLISHED_CHOICES = (
+    (PRIVATE, 'Private'),
+    (SHARED, 'Shared'),
+    (PUBLIC, 'Public')
+    )
+
+    def user_directory_path(instance, filename):
+        return 'user_{0}/{1}'.format(instance.user.id, filename)
+
+    photo_id = models.UUIDField(
+               primary_key=True,
+               default=uuid.uuid4,
+               editable=False)
+
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
Instead of this you could just import `User` from `django.contrib.auth.models` and use that for the photos and albums